tests/remote/test_client.py

```
@@ -0,0 +1,316 @@
+        _status(False, 'Job has no running steps.'))
+
+    assert running is False
+    assert completed == []
+    assert starttimes == {}
+
+
+def test_report_job_status_unparsable_message(gcd_nop_project, caplog):
+    '''A message that is not the node payload is reported, not crashed on'''
+    client = _client(gcd_nop_project)
+
+    completed, starttimes, running = client._report_job_status(
```

Ruff (0.16.1)

[warning] Unpacked variable `starttimes` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)